src/kmer.jl

```diff
@@ -167,7 +168,8 @@ function Kmer{A,K,N}(itr) where {A,K,N}
         Base.@_inline_meta
         body = zero(UInt64)
         @inbounds for i in 1:n_per_chunk
```

Comment:
Not critical, but I think `for i in 1:x` is starting to be considered gauche due to things like offset arrays. Is there a way to do this with `firstindex` or similar (only if it doesn't require bending over backwards).